src/scripts/main.js

```diff
@@ -1 +1,56 @@
 'use strict';
+
+function showNotification(message, type) {
+  const notification = document.createElement('div');
+
+  notification.setAttribute('data-qa', 'notification');
+  notification.className = type;
+  notification.textContent = message;
+  document.body.appendChild(notification);
+}
+
+const firstPromise = new Promise(function (resolve, reject) {
+  document.addEventListener('click', () => {
+    clearTimeout(timer);
+    resolve('First promise was resolved');
+  });
+
+  const timer = setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+firstPromise
+  .then((message) => showNotification(message, 'success'))
+  .catch((error) => showNotification(error, 'error'));
+
+const secondPromise = new Promise(function (resolve, reject) {
+  document.addEventListener('click', (events) => {
+    if (events.button === 0 || events.button === 2) {
```

Comment:
The condition 'events.button === 0 || events.button === 2' checks for left and right mouse clicks. Ensure this logic aligns with the intended functionality, as it resolves the promise on either click.